bentoml/artifact/tf_keras_model_artifact.py

```diff
@@ -68,3 +69,30 @@ def save(self, dst):
 
     def get(self):
         return self._model
+
+
+class _TfKerasModelWrapper(object):
+
+    def __init__(self, model):
+        try:
+            import tensorflow as tf
+            from tensorflow.python.keras.models import load_model
+        except ImportError:
+            raise ImportError("tensorflow package is required to use TfKerasModelArtifact")
+        self.session = tf.Session()
+        self.graph = tf.get_default_graph()
+        with self.graph.as_default():
+            with self.session.as_default():
```

Comment:
is this necessary?

Reply:
I think so, you need to load the model in the same session/graph